less/about/error.less

```diff
@@ -1,4 +1,8 @@
-@import "./common.less";
```

Comment:
deprecated thanks to https://github.com/brave/browser-laptop/pull/10881/files#diff-2369573669cb068e3f220a1c4ef7090cR13 and https://github.com/brave/browser-laptop/pull/10881/files#diff-103b59b981aa5f02b62b3f712ca69c19R12